srcpkgs/python3-rapidfuzz/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'python3-rapidfuzz'
+pkgname=python3-rapidfuzz
+version=3.2.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="cmake python3-scikit-build"
+makedepends="python3-devel rapidfuzz-cpp"
+depends="python3"
+checkdepends="python3-hypothesis python3-pandas python3-pytest"
+short_desc="Rapid fuzzy string matching in Python using various string metrics"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="MIT"
+homepage="https://github.com/maxbachmann/rapidfuzz"
+distfiles="${PYPI_SITE}/r/rapidfuzz/rapidfuzz-${version}.tar.gz"
+checksum=448d031d9960fea7826d42bd4284156fc68d3b55a6946eb34ca5c6acf960577b
+
+pre_build() {
+	# ugly hack to help cmake find python3 headers
+	if [ "$CROSS_BUILD" ]; then
+		rm -rf ${XBPS_BUILDDIR}/../${py3_inc}
+		ln -sf ${XBPS_BUILDDIR}/../${XBPS_CROSS_BASE}/${py3_inc} \
+			${XBPS_BUILDDIR}/../${py3_inc}
+	fi
+}
```

Comment:
don't do this, there should be a cmake flag that can be passed to tell it where things are.

`zap` and re-`binary-bootstrap` your masterdirs

Reply:
@classabbyamp done